projects/ngx-auth-firebaseui/src/lib/components/ngx-auth-firebaseui/auth.component.html

```diff
@@ -21,23 +21,21 @@
     </ngx-auth-firebaseui-email-confirmation>
   </div>
 
-  <ng-template #signedInUser>
-    <div class="signed-in-container" fxLayout="column" fxLayoutAlign="center center">
-      <img *ngIf="user?.photoURL; else noPhoto" [src]="user?.photoURL" class="account-circle">
-      <ng-template #noPhoto>
-        <mat-icon class="account-circle">account_circle</mat-icon>
-      </ng-template>
-      <div class="user-display-name mat-title">{{ user?.displayName }}</div>
-      <div class="user-email mat-body-2">{{ user?.email }}</div>
-      <div class="actions">
-        <mat-progress-bar *ngIf="isLoading" mode="indeterminate"></mat-progress-bar>
-        <a [routerLink]="goBackURL" class="go-back-button action-button" color="primary"
-           mat-stroked-button>{{ verifyEmailGoBackText }}</a>
-        <button (click)="signOut()" class="sign-out-button action-button" color="warn"
-                mat-stroked-button>{{ signOutText }}</button>
-      </div>
+  <div *ngIf="isUserProfileScreenVisible(user)" class="signed-in-container" fxLayout="column" fxLayoutAlign="center center" *ngIf="">
```

Comment:
can u please provide more info about the implementation and why the logic should change ?

Reply:
In current implementation, only two "alternatives" are covered:
 - If the user is signed In: we display the profile or if the email is not verified (and guard exists) then we display the panel telling the user must validate its email.
 - If the user is not signed in: we display the form

My proposal is to offer a third option (implementation detail: I moved the condition in a method so the HTML is more readable).

To allow this third option i removed the `if ; else` syntax, thus removing the ng-template. After my change we basically have 3 options:
 - If the user is not connected: display the form
 - If the user is connected and doesn't have email validated (and a guard exists) we display the email validation screen
 - If the user is connected and have email validated but for a reason I don't want, where I placed the form, to display the "profile" data (picture, logout button etc), then we add a flag to allow not to display this profile. This "profile" view replacing the form after login may disturb the design.

Since everything has parameters in ngx-auth-firebaseui, it's nice to allow developper not to display something and allowing him to use the new ngx-auth-firebaseui with signin/register/email validation guard without forcing display of profile after login.

Reply:
I will perform more tests to find if there is another way to achieve this :)